vendor.conf

```
@@ -1,3 +1,4 @@
+cloud.google.com/go ac3720150beed4ebd66918aa31d4713110098260
```

I see they tag releases https://github.com/googleapis/google-cloud-go/releases; can we use the commit of a tagged version for this, and add a comment to mention that version? e.g.

```suggestion
cloud.google.com/go 0ebda48a7f143b1cce9eb37a8c1106ac762a3430 # v0.34.0
```

(current version looks to be take directly from master, and is 29 commits ahead of v0.34.0 https://github.com/googleapis/google-cloud-go/compare/v0.34.0...ac3720150beed4ebd66918aa31d4713110098260)

---

Actually; I see compose-on-kubernetes uses that version as well https://github.com/docker/compose-on-kubernetes/commit/be6d5be2f61bdfdc0716a5c50393e4f4720e8800#diff-bd247e83efc3c45ae9e8c47233249f18R18

---

Looks like you forgot to update this to the version (v0.3.4) that's used by compose-on-kubernetes ( see my [earlier comment](https://github.com/docker/cli/pull/1583/files#r243203663); https://github.com/docker/compose-on-kubernetes/blob/6347392617e1efd3077f8a918d67fa94cd39289e/Gopkg.lock#L12-L18